direct-hs/src/Web/Direct/DirectRPC.hs

```diff
@@ -115,6 +115,14 @@ createPairTalk rpcclient dom peer = do
     rsp <- callRpcThrow rpcclient methodName dat
     convertOrThrow methodName (decodeTalkRoom [peer]) rsp -- fixme: users
 
+deleteTalker :: RPC.Client -> TalkRoom -> User -> IO ()
+deleteTalker rpcclient talk user = do
+    let methodName = "delete_talker"
+        tid        = talkId talk
+        uid        = userId user
+        dat        = [M.ObjectWord tid, M.ObjectWord uid]
+    void $ callRpcThrow rpcclient methodName dat
```

Comment:
I don't recommend rethrowing an exception here.
I guess removing `void` here makes this function return `Either Exception ()`, which is compatible with `removeUserFromTalkRoom`.

As the general policy, I decided that at least any errors returned by direct4b.com via RPC must be explicitly returned as `Left`. Actually, most value constructors of the `Exception` correspond to the error returned by direct4b.com.

Reply:
Sorry I've made one mistake: you have to use `callRpc` instead of `callRpcThrow` in addition to removing `void`.